2-repositories/iglu-server/src/main/scala/com/snowplowanalytics/iglu/server/Server.scala

```diff
@@ -97,7 +97,7 @@ object Server {
   def run(config: Config)(implicit ec: ExecutionContext, cs: ContextShift[IO], timer: Timer[IO]): Stream[IO, ExitCode] = {
     for {
       client <- BlazeClientBuilder[IO](ec).stream
-      webhookClient = Webhook.WebhookClient(config.webhooks.getOrElse(List.empty), client)
+      webhookClient = Webhook.WebhookClient(config.webhooks.getOrElse(Config.Webhooks(None)).schemaPublished.getOrElse(List()), client)
```

Comment:
The whole point of `WebhookClient` is that it has *all* kinds of webhooks and decides itself when to fire each type. I know that we don't have any other right now, but once we get them - somebody will forget to remove this `schemaPublished`